src/Pos/Wallet/Web/Server/Full/R.hs

```diff
@@ -52,9 +54,11 @@ runWRealMode
 runWRealMode db conn =
     runRealBasedMode
         unwrapWPMode
-        (lift . lift)
+        -- doesn't work
+        -- (powerLift @(RealMode WalletSscType) @WalletRealWebMode)
+        (liftWalletRedirects . lift . lift)
```

Comment:
`liftWalletRedirects` is just an alias for `coerce`. 
`WalletWebSockets` is `Ether.ReaderT' ConnectionsVar`. `WalletWebDB` is `Ether.ReaderT' WalletState`.

I can't use `coerce` to lift something into reader.